docs/source/developer/contributing.rst

```diff
@@ -908,17 +908,17 @@ Or with ``jlpm``:
 Writing Style
 ^^^^^^^^^^^^^
 
--  The documentation should be written in the second person, referring
-   to the reader as "you" and not using the first person plural "we."
+-  Write documentation in the second person, referring
+   to the reader as "you". Do not use the first person plural "we."
    The author of the documentation is not sitting next to the user, so
    using "we" can lead to frustration when things don't work as
    expected.
 -  Avoid words that trivialize using JupyterLab such as "simply" or
    "just." Tasks that developers find simple or easy may not be for
    users.
--  Write in the active tense, so "drag the notebook cells..." rather
-   than "notebook cells can be dragged..."
--  The beginning of each section should begin with a short (1-2
+-  Write in the active tense. Use "drag the notebook cells…" rather
```

Comment:
Rather than breaking the sentence, would it make sense to use e.g.,?

Write in the active tense e.g., use "drag the notebook cells.." rather than "notebook cells can be dragged..."

Reply:
I prefer "For example" to "e.g." for clarity. Microsoft's style guide also prefers "for example" or "examples": https://docs.microsoft.com/en-us/style-guide/procedures-instructions/writing-step-by-step-instructions